scripts/ps.py

```diff
@@ -17,17 +17,28 @@
 
 import psutil
 
+PROC_STATUSES_RAW = {
+    psutil.STATUS_RUNNING: "R",
+    psutil.STATUS_SLEEPING: "S",
+    psutil.STATUS_DISK_SLEEP: "D",
+    psutil.STATUS_STOPPED: "T",
+    psutil.STATUS_TRACING_STOP: "t",
+    psutil.STATUS_ZOMBIE: "Z",
+    psutil.STATUS_DEAD: "X",
+    psutil.STATUS_WAKING: "W"
+}
```

Comment:
I updated these:
psutil.STATUS_IDLE(OSX, FreeBSD):		I
psutil.STATUS_LOCKED(FreeBSD):		L
psutil.STATUS_WAITING(FreeBSD):			W
psutil.STATUS_SUSPENDED(NetBSD):			V

But:
psutil.STATUS_WAKE_KILL
psutil.STATUS_WAKING
the ps cmd of Linux did not take into account.

Maybe we should do the same thing.